index.html

```diff
@@ -10,50 +10,61 @@
     <script src="https://kit.fontawesome.com/7415ea4535.js" crossorigin="anonymous"></script>
 <body>
 <div class="container">
```

Comment:
- [ ] Kindly make this container `100vw`, it's not full screen at some screen sizes 👇 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/213429314-569315c2-9c07-4de9-bfbe-a0f96cfd865c.png)